abstract/TypedData.js

```
@@ -1,71 +1,128 @@
   get uid() {
     return this.__ctxId;
   }
 
   /**
-   * @param {String} prop
-   * @param {any} value
+   * @param {ExtractKeysFromSchema<T>} prop
+   * @param {ExtractDataFromSchema<T>[prop]} value
    */
   setValue(prop, value) {
     if (!this.__typedSchema.hasOwnProperty(prop)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)